app/src/sharedTest/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsFragmentTest.kt

```diff
@@ -97,6 +97,7 @@ class StatisticsFragmentTest {
             .getString(R.string.statistics_active_tasks, 50.0f)
         val expectedCompletedTaskText = getApplicationContext<Context>()
             .getString(R.string.statistics_completed_tasks, 50.0f)
+        Thread.sleep(4000)
```

Comment:
Remove this - I think it got in here by accident.

Reply:
Missed this! Again!